src/game/ctrl.rs

```diff
@@ -122,27 +121,34 @@ impl GameController {
                     }
                 }
                 true
-            },
+            }
             Gesture::NoGesture => false,
-            _ => { panic!("Unimplemented gesture type."); }
+            _ => {
```

Comment:
Could you please remove the braces and move the panic to the same line as the `=>`?